pkg/gits/LICENSE

```diff
@@ -0,0 +1 @@
+TODO ;-)
```

Comment:
Was this added by mistake? Why would we have a license file here?

Reply:
Yeah.